runtime/logging/logger.go

```diff
@@ -129,41 +129,51 @@ func StderrLogger(opts Options) *slog.Logger {
 	return slog.New(&LogHandler{Opts: opts, Write: writeText})
 }
 
-// testLogger implements a logger for tests.
-type testLogger struct {
+// TestLogger implements a logger for tests.
```

Comment:
Perhaps not for this change, but I wonder if we can avoid placing a dependency on the testing package from a non test file somehow?

One option would be to depend on a smaller interface than testing.TB (do we need more than the Log() method?). Another option (perhaps infeasible) would be to move this code to the weavertest package.